### PR TITLE
Feature/5 enable orthanc explorer 2

### PR DESCRIPTION
**Pull request by exico91**
Added explicit settings and configure orthanc explorer 2
Added a version lock

### CODE REVIEW

File: docker/orthanc/orthanc.json [hunk 1]
```diff
@@ -42,5 +42,71 @@
...
+      "StudyListSearchAsYouTypeMinChars": 3,
+      "StudyListSearchAsYouTypeDelay": 400,
+      "StudyListContentIfNoSearch": "most-recents",
+      "EnableDicomModalities": true,
+      "EnableDownloadZip": true,
+      "EnableAnonymization": false,
+      "EnableSendTo": true,
+
+      "ModalitiesFilter": [
+        "CR", "CT", "DX", "MG", "MR", "NM", "OT", "PT", "PX", "SEG", "SR", "US", "XA"
```
Comment (zgypa) [inline docker/orthanc/orthanc.json]: Le modalities da filtrare sono:
* RG
* PX
* PR
* OSS
* M3D
* MR
* LS
* IO
* DX
* DOC
* CT
* XC

File: docker/orthanc/orthanc.json [hunk 2]
```diff
@@ -42,5 +42,71 @@
...
+      ],
+      "SeriesMainTags" : [],
+      "MaxStudiesDisplayed": 100,
+      "StudyListSearchMode": "search-as-you-type",
+      "StudyListSearchAsYouTypeMinChars": 3,
+      "StudyListSearchAsYouTypeDelay": 400,
+      "StudyListContentIfNoSearch": "most-recents",
+      "EnableDicomModalities": true,
+      "EnableDownloadZip": true,
+      "EnableAnonymization": false,
```
Comment (zgypa) [inline docker/orthanc/orthanc.json]: Why disable Anonymization? Did we discuss this?

**Review by zgypa — COMMENTED** [review]
See comments.

File: docker/orthanc/orthanc.json (hunk 2 above)
Comment (exico91) [reply docker/orthanc/orthanc.json]: mmh I believe that is a misconfiguration, should be true

File: docker/orthanc/orthanc.json (hunk 1 above)
Comment (exico91) [reply docker/orthanc/orthanc.json]: I tough i copied them , Ill edit the config immediately

**Review by zgypa — APPROVED** [review]

**Pull request merged**